src/NuGet.Server/Infrastructure/ServerPackage.cs

```diff
@@ -79,8 +39,10 @@ public ServerPackage(IPackage package, PackageDerivedData packageDerivedData)
             MinClientVersion = package.MinClientVersion;
             ReportAbuseUrl = package.ReportAbuseUrl;
             DownloadCount = package.DownloadCount;
-            IsAbsoluteLatestVersion = package.IsAbsoluteLatestVersion;
-            IsLatestVersion = package.IsLatestVersion;
+            SemVer1IsAbsoluteLatest = false;
+            SemVer1IsLatest = false;
+            SemVer2IsAbsoluteLatest = false;
+            SemVer2IsLatest = false;
```

Comment:
If I remember right these flags existed because OData will sort the versions as strings. So to find the actual latest version using correct semver sort it was easier for OData to just filter on these bools.

Maybe that issue has been fixed, if it has we should look into just getting rid of these bools and calculating it as part of the request.